R/recipe.R

```diff
@@ -730,7 +730,9 @@ print.recipe <- function(x, form_width = 30, ...) {
 #' @param ... further arguments passed to or from other methods (not currently
 #'   used).
 #' @return A tibble with columns `variable`, `type`, `role`,
-#'   and `source`.
+#'   and `source`. When `original = TRUE`, an additional column is included
+#'   named `required to bake` (based on the results of
```

Comment:
Maybe `required_to_bake`? I try to avoid spaces in names, but if the `summary()` method is purely for interactive exploration, maybe it is okay